X/.i3/config

```diff
@@ -1,4 +1,4 @@
-# This file has been auto-generated by i3-config-wizard(1).
+# This file has been auto-generated by i1-config-wizard(1).
```

Comment:
`i1` ?

Reply:
heh, vim... must have lowered that on accident, will patch